platform_api/orchestrator/kube_client.py

```diff
@@ -2631,8 +2648,8 @@ class KubePreemption:
     def get_pods_to_preempt(
         cls, resources: NodeResources, pods: list[PodDescriptor]
     ) -> list[PodDescriptor]:
-        if resources.gpu:
-            pods = [p for p in pods if p.resources and p.resources.gpu]
+        if resources.nvidia_gpu:
```

Comment:
we probably should do similar filtering for AMD too, wdyt?

Reply:
done